source/code/plugins/filter_hdinsight.rb

```diff
@@ -11,24 +11,27 @@ class HdinsightFilter < Filter
     BASE_DIR = File.dirname(File.expand_path('..', __FILE__))
     RUBY_DIR = BASE_DIR + '/ruby/bin/ruby '
     SCRIPT = BASE_DIR + '/bin/hdinsightmanifestreader.rb '
-	
-	attr_accessor :command
+
+    attr_accessor :command
 
     def configure(conf)
       super
-      @hostname = OMS::Common.get_hostname or "Unknown host"
+      @hostname = OMS::Common.get_hostname or "unknown"
```

Comment:
Any reason that this string changed? "unknown" is a little less clear than "Unknown host"